### ISSUE TITLE
Feature: Add launch button to details view of job template

### ISSUE THREAD

**Issue by kdelee** [ansible/awx#2709]
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME

 - UI

##### SUMMARY

When I click on the detail of a job temlate/workflow job template in the Templates view, the editable details appear at the top of the page. If I have multiple pages of templates and this one does not happen to be on the first page, there is no way for me to launch the template from this page. I can look through the later pages to find the item in the list and find the launch button (looks like a rocket) or I can copy the name and put that in the search bar to narrow the list down to the job template.

TL; DR that is a lot of work to launch the template I was just working on.

Here is a screen shot with some suggested places to add it. Hopefully better looking rendition...

![no_launch_button_job_template_detail_possible_places_to_add](https://user-images.githubusercontent.com/29239279/48636336-99d56500-e998-11e8-8cc0-52647615b4f6.png)

**Comment by wenottingham** [comment]
-> #1991
